sdk/tests/conformance/context/00_test_list.txt

```diff
@@ -1,6 +1,7 @@
 --max-version 1.9.9 constants-and-properties.html
 --min-version 1.0.2 context-attribute-preserve-drawing-buffer.html
 context-attributes-alpha-depth-stencil-antialias.html
+context-size-change.html
```

Comment:
Please add --min-version 1.0.4 as a prefix (it's just the way new tests are added at this point).